walletlibrary/src/test/java/com/microsoft/walletlibrary/did/sdk/RevocationServiceTest.kt

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
Can we just delete this and the IdentifierManager files that are completely commented out?